misc/ballerinalang-data-mapper/src/main/java/org/ballerinalang/datamapper/config/LSClientExtendedConfig.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ballerinalang.datamapper.config;
+
+import org.ballerinalang.langserver.compiler.config.LSClientConfig;
+
+/**
+ * Extended Ballerina Client Configuration.
+ *
+ * @since 2.0.0
+ */
+public class LSClientExtendedConfig extends LSClientConfig {
```

Comment:
Shall we rename this to `DatamapperClientConfig` or similar one, without using the extended word?

Reply:
Let's fix this in a separate PR.